solr/solrj/src/test/org/apache/solr/common/util/ContentStreamTest.java

```diff
@@ -161,7 +161,7 @@ public void testURLStreamCSVGZIPExtention() throws IOException {
     }
 
     ContentStreamBase stream =
-        new ContentStreamBase.URLStream(new URL(file.toURI().toASCIIString()));
+        new ContentStreamBase.URLStream(URI.create(file.toURI().toASCIIString()).toURL());
```

Comment:
this code makes no sense (also at other places above). Just call `file.toURI().toURL()`

Reply:
see other occurrences of this in this and other files